server/run-tests.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+LOG_LEVEL="test"
+
+while [[ "$#" -gt 0 ]]; do
+  case $1 in
+  --debug)
+    LOG_LEVEL="debug"
+    shift
+    ;;
+  esac
+  shift
+done
+
+export NODE_ENV="test"
+export SYNC_DATABASE="false"
+export SQL_DATABASE="dds_test"
+export LOG_LEVEL="$LOG_LEVEL"
+
+# Create test database if it doesn't exist.
+if [ "$(psql -tAc "SELECT 1 FROM pg_database WHERE datname='$SQL_DATABASE'" -h "$SQL_HOST" -p "$SQL_PORT" -U "$SQL_USER")" != '1' ]; then
```

Comment:
When attempting to run this PR I ran into some strange output from this command. It created the db successfully the first time, but it doesn't look like this check is entirely correct.
`psql: error: FATAL:  database "jwileczek" does not exist
Creating 'dds_test' database... createdb: error: database creation failed: ERROR:  database "dds_test" already exists
failed to create 'dds_test' database. Aborting...`
It looks like the initial check for the dds_test database isn't always correct. I changed it to use this method of checking if that db exists that works for me:
`if [ "$(psql -lqt | cut -d \| -f 1 | grep -qw $SQL_DATABASE)"];`